threshold_tools_event_types.ipynb

```diff
@@ -0,0 +1,505 @@
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "hourly_data_subset = app.load(hourly_data_subset)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<a id='transform'></a>\n",
+    "## Step 2: Get block maxima series for different event types\n",
+    "\n",
+    "To facilitate custom definitions of different types of extreme events, there are optional keyword arguments to the `get_block_maxima` function to pull the maximum annual value. The keyword options are:\n",
+    "- `duration`: how long an event continuously lasts in hours (see example 2)\n",
+    "- `groupby` and  `grouped_duration`: work together to define multi-day events (see examples 3 and 4)\n",
+    "\n",
+    "The following four examples show how to use these options to construct different types of events of interest. See section 3 for further analysis and discussion of these events."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Example 1: hottest hour in each year\n",
+    "The basic use case of `get_block_maxima` pulls the maximum value in each year of data, shown here:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "ams = get_block_maxima(hourly_data_subset)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "ams.plot()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Example 2: the hottest continuous 3-hour event in each year\n",
+    "Users may be interested in extremes that last longer than 1 hour. This example identifies the temperature value that corresponds to the hottest continuous 3-hour period in each year, using the optional `duration` argument. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "ams_3h = get_block_maxima(\n",
+    "    hourly_data_subset,\n",
+    "    duration = (3, 'hour')\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "ams_3h.plot()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Example 3: the hottest temperature that is reached 3 days in a row\n",
+    "This example identifies the max temperature value that is reached for at least 1 hour each day for 3 days in a row, using the `groupby` and `grouped_duration` arguments."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "ams_3d = get_block_maxima(\n",
+    "    hourly_data_subset,\n",
+    "    groupby = (1, 'day'), \n",
+    "    grouped_duration = (3, 'day')\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Note: effective sample size\n",
```

Comment:
Ah this is really helpful! I would suggest just moving this up in the notebook to when the user might first encounter the warning message